bennu-toolkit/src/main/webapp/bennu-toolkit/js/spinner.js

```diff
@@ -0,0 +1,48 @@
+(function(){
+    Bennu.spinner = Bennu.spinner || {};
+
+    Bennu.spinner.attr = "bennu-spinner";
+
+    Bennu.spinner.createWidget = function(e){
+      e = $(e);
+      var template = "<i class='bennu-spinner-widget icon icon-spin'></i>";
```

Comment:
Since the toolkit already embeds FontAwesome, shouldn't we use its spinner until Canvas is fully standardized?